website/config/routes.js

```diff
@@ -499,6 +499,7 @@ module.exports.routes = {
   'GET /fleetctl-preview':   '/try-fleet/fleetctl-preview',
   'GET /try-fleet/sandbox-expired':   '/try-fleet/fleetctl-preview',
   'GET /try-fleet/sandbox':   '/try-fleet/fleetctl-preview',
+  'GET /docs/using-fleet/mdm-macos-updates': '/learn-more-about/os-updates',
```

Comment:
There should be as few distinct route targets as possible (RHS)

Reply:
I think we want this one for backwards compatibility. I moved it to the backwards compat section.

The UI in old Fleet versions will link to `/docs/using-fleet/mdm-macos-updates`.

Pointing that to the solid `/learn-more-about/os-updates` route so we have the flexibility to move the UI link to another location.